lib/metasploit/framework/login_scanner/base.rb

```diff
@@ -247,8 +261,8 @@ def scan!
                 if result.status == Metasploit::Model::Login::Status::UNABLE_TO_CONNECT
                   consecutive_error_count += 1
                   total_error_count += 1
-                  break if consecutive_error_count >= 3
-                  break if total_error_count >= 10
+                  break if consecutive_error_count >= maxconsecutiveerrors
```

Comment:
Thanks for the PR; Does this change break other scanners? i.e. the other scanners will be set to `nil` here and crash

If they do crash - I believe we need to set sane default values here https://github.com/rapid7/metasploit-framework/blob/d82c91a9a49590f9ad700fa80238fda7b7b3b930/lib/metasploit/framework/login_scanner/base.rb#L307-L314